src/sentry/api/serializers/models/group.py

```diff
@@ -742,12 +742,28 @@ class GroupSerializerSnuba(GroupSerializerBase):
         "first_seen",
         "last_seen",
         "times_seen",
+        "date",  # We merge this with start/end, so don't want to include it as its own
+        # condition
     }
 
     def __init__(self, environment_ids=None, start=None, end=None, search_filters=None):
+        from sentry.search.snuba.executors import get_search_filter
+
         self.environment_ids = environment_ids
-        self.start = start
-        self.end = end
+
+        # XXX: We copy this logic from `PostgresSnubaQueryExecutor.query`. Ideally we
+        # should try and encapsulate this logic, but if you're changing this, change it
+        # there as well.
+        self.start = None
+        start_params = [start, get_search_filter(search_filters, "date", ">")]
```

Comment:
Need to check whether search_filters is None here, happens when the query bar is cleared out